src/en/disasterscans/src/eu/kanade/tachiyomi/extension/en/disasterscans/DisasterScans.kt

```diff
@@ -1,209 +1,133 @@
 
-        val chapterPages = document.select("#__NEXT_DATA__").html()
-            .parseAs<NextData<ApiChapterPages>>()
-            .props.pageProps.chapter.pages
+        val infoBlock = document.selectFirst("section div div")
 
-        val pages = chapterPages.parseAs<List<String>>()
+        if (infoBlock != null) {
+            val infoBlockChildren = infoBlock.children()
 
-        val cdnUrl = updatedCdnUrl(document)
+            infoBlockChildren[0].selectFirst("h1")?.text()?.let { title = it }
+            description = infoBlockChildren[2].text()
 
-        return pages.mapIndexed { idx, image ->
-            Page(idx, "", "$cdnUrl$image")
-        }
-    }
-
-    private fun updatedCdnUrl(document: Document): String {
-        val cdnUrlFromPage = document.selectFirst("main div.maxWidth img")
-            ?.attr("src")
-            ?.substringBefore("?")
-            ?.let { "$it?fileId=" }
-
-        return preferences.getCdnUrl()
-            .let {
-                if (it != cdnUrlFromPage && cdnUrlFromPage != null) {
-                    preferences.putCdnUrl(cdnUrlFromPage)
-                    cdnUrlFromPage
-                } else {
-                    it
+            with(infoBlockChildren[1].select("span")) {
+                status = when (this.removeAt(0)?.text()?.lowercase()) {
+                    "ongoing" -> SManga.ONGOING
+                    else -> SManga.UNKNOWN
                 }
+                genre = this.joinToString(", ") { it.text() }
             }
+        }
     }
 
-    private inline fun <reified T> String.parseAs(): T =
-        json.decodeFromString(this)
-
-    private inline fun <reified T> Response.parseAs(): T =
-        body.string().parseAs()
-
-    private fun SharedPreferences.getCdnUrl(): String {
-        return getString(cdnPref, fallbackCdnUrl) ?: fallbackCdnUrl
-    }
-
-    private fun SharedPreferences.putCdnUrl(url: String) {
-        edit().putString(cdnPref, url).commit()
-    }
+    @Serializable
+    class ChapterDTO(val chapterID: Int, val ChapterNumber: String, val ChapterName: String, val chapterDate: String)
+    private val chapterDataRegex = Regex("""\\"chapters\\":(\[.*]),\\"param\\":\\"(\S+)\\"\}""")
 
-    companion object {
-        private const val fallbackCdnUrl = "https://f005.backblazeb2.com/b2api/v1/b2_download_file_by_id?fileId="
-        private const val cdnPref = "cdn_pref"
-        val titleSpecialCharactersRegex = "[^a-z0-9]+".toRegex()
-        val trailingHyphenRegex = "-+$".toRegex()
-        val dateFormat by lazy {
-            SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", Locale.ENGLISH)
+    override fun chapterListParse(response: Response): List<SChapter> {
+        val (chapterData, mangaId) = chapterDataRegex.find(response.body.string())
+            ?.destructured ?: return emptyList()
+
+        return json.decodeFromString<List<ChapterDTO>>(chapterData.replace("\\", "")).map { chapter ->
```

Comment:
See if the approach in VortexScans works for this use case too:
https://github.com/keiyoushi/extensions-source/blob/64ea3602c060586ebfcb76cd8388b04b3d5b43b3/src/en/arvenscans/src/eu/kanade/tachiyomi/extension/en/arvenscans/VortexScans.kt#L27-L32